src/second-iter.md

```diff
@@ -448,10 +448,10 @@ cargo build
 
 🎉 🎉 🎉
 
-The as_deref and as-derf_mut functions are stable as of Rust 1.40. Before that you
+The as_deref and as_derf_mut functions are stable as of Rust 1.40. Before that you
```

Comment:
This should be `as_deref_mut` actually.

Reply:
Oops, you're quite right, I missed that missing letter.